src/components/Common/HelpCard.tsx

```diff
@@ -16,6 +16,7 @@ function HelpCard({ title, content, icon, theme, link }: Props) {
       href={link}
       target="_blank"
       rel="noopener noreferrer"
+      data-name={`help-card-${theme}`}
```

Comment:
I think we can improve the readability here. Rather than using theme, a data-name attribute can be passed from parent and used here.
```suggestion
      data-name={data-name}
```

You can make this change in

Reply:
Makes sense, I'll work on this change today

Reply:
Ok I've push the changed you requested, let me know if there's anything else that should be updated